### PR TITLE
Adding support for wdio-mochawesome-reporter

### PR DESCRIPTION
**Pull request by fijijavis**
## Proposed changes

This change will add support for the new wdio-mochawesome-reporter to the wdio CLI

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: lib/cli.js [hunk 1]
```diff
@@ -257,6 +258,12 @@ WDIO Configuration Helper
         default: './',
         when: (answers) => answers.reporters === 'json'
     }, {
+        type: 'input',
+        name: 'outputDir',
+        message: 'In which directory should the mochawesome json reports get stored?',
+        default: './',
+        when: (answers) => answers.reporters === 'mochawesome'
```
Comment (christian-bromann) [inline lib/cli.js]: This is not quite right, I think `answers.reporters` is an array. Do you mind updating the two example above of this too?

Comment (christian-bromann) [reply lib/cli.js]: Feel free to write
```js
when: (answers) => answers.reporters.includes('mochawesome')
```

Comment (fijijavis) [reply lib/cli.js]: Done

**Comment by fijijavis** [comment]
You're welcome.  Hopefully folks find it useful.

**Pull request merged**